src/co_tools/co_fastq.py

```diff
@@ -0,0 +1,156 @@
+import os
+import re
+import subprocess
+import sys
+from glob import glob
+from pathlib import Path
+
+# local imports
+from .get_logger import LOGGER
+
+
+def get_fastq_pair(dir_path: str = "../data"):
+    total_dirs = 0
+    for base, dirs, files in os.walk(dir_path):
+        for dir in dirs:
+            total_dirs += 1
+    if total_dirs != 2:
+        LOGGER.error(
+            f"The fastq files in {dir_path} are not properly configured"
+            + " to use this function. There should be only 2 folders"
+            + " inside the data folder."
+        )
+        return 0
+    prefix_dict, this_prefix = {}, None
+    fwd, rev = None, None
+    for path in glob(str(f"{dir_path}/**/*.fastq.gz"), recursive=True):
+        if prefix := get_prefix(path):
```

Comment:
Maybe I'm missing something but why is this := instead of = ? It doesn't seem like we're using it as part of the if statement, is it just checking for an empty string?

Reply:
`:=` allows you to assign a value to a variable and then use that variable in the `if` statement. Without using `:=` you would need a separate line of code to first assign the value of the variable. This simplifies code.

Reply:
Yes, but what are you checking?

Reply:
`prefix` is being checked. If `prefix` evaluates to False then you move on. Using `:=` allows you the ability to assign a value to a variable and check how it evaluates in a single line of code. It's called `assignment expressions` in Python
https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Reply:
Yes, but when does prefix evaluate to false? That's what I'm trying to get at.

Reply:
I need to refactor get_prefix() so that it returns `0` when no underscores are in the filename. Then this particular assignment expression will have the ability to evaluate to False. I'll update shortly.

Reply:
You can just set it to False directly too :)